rust/core-lib/src/editor.rs

```diff
@@ -1137,9 +1113,6 @@ impl Editor {
             RequestLines(LineRange { first, last }) => self.do_request_lines(first, last),
             Yank => self.yank(),
             Transpose => self.do_transpose(),
-            Click(MouseAction {line, column, flags, click_count} ) => {
```

Comment:
This will break all frontends that aren't xi-mac, so maybe we don't want to remove it right away? What would be ideal is if we can keep it in place and just call `do_gesture` with the appropriate gesture for a given click event.

We could also consider logging a deprecation warning, maybe pointing people to this PR?

This actually reminds me that, ideally, this PR would include updates to the docs to include the newly added gestures. I'd been forgetting this, we've had a few recent PRs without doc updates.

Reply:
That makes sense. I will leave it there for now to prevent other clients from breaking.

I will add it to the documentation.